test_main.py

```diff
@@ -106,3 +106,16 @@ def control_freezers():
   assert db.delete_freezer(1,4) == True
   assert db.get_freezers(1) == [(3, 'Booh'), (5, 'Hooker')]
   reset.reset(True)
+
+def mexican():
```

Comment:
tests won't be run if they don't start with `test_`, I believe

Reply:
Seems like #67 confirms this. Alright, lemme close that pull request and get this fixed.
(How did you not notice that earlier? There's multiple functions in there that do not start with `test_`!